packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -126,21 +126,21 @@ const resolveConfigFiles = async (args) => {
         const defaultConfig = configFiles.find((p) => p.path.includes(mode) || p.path.includes(modeAlias[mode]));
 
         if (defaultConfig) {
-            opts = await finalize(defaultConfig, args);
+            opts = await finalize(defaultConfig, args, true);
             return;
         }
 
         const foundConfig = configFiles.pop();
 
-        opts = await finalize(foundConfig, args);
+        opts = await finalize(foundConfig, args, true);
 
         return;
     }
 };
 
 // Given config data, determines the type of config and
 // returns final config
-const finalize = async (moduleObj, args) => {
+const finalize = async (moduleObj, args, isDefault = false) => {
```

Comment:
```suggestion
const finalize = async (moduleObj, args, isDefaultConfig = false) => {
```

Reply:
after ci

Reply:
@anshumanv Can you fix it, I want to merge it and do release

Reply:
fixed